lib/ansible/modules/cloud/amazon/aws_kms.py

```diff
@@ -1004,6 +1028,7 @@ def main():
         policy=dict(),
         purge_grants=dict(type='bool', default=False),
         state=dict(default='present', choices=['present', 'absent']),
+        enable_key_rotation=(dict(type='bool', default=False))
```

Comment:
I would prefer that this defaulted to None, otherwise someone might find their existing key rotation suddenly gets disabled when they just want to tweak the tags.

Reply:
Ok, true. I change this.

Reply:
maybe the two lines

```
    current_rotation_status = connection.get_key_rotation_status(KeyId=key_id)
    result['enable_key_rotation'] = current_rotation_status.get('KeyRotationEnabled')
```

should also be appended to the `aws_kms_info` module.